docs/faststore/docs/headless-cms/multiple-page-template.mdx

```diff
@@ -86,18 +74,38 @@ The system first searches for a template named after the product slug. For examp
 
 ![pdp-page-list](https://vtexhelp.vtexassets.com/assets/docs/src/pdplist___d6f00d8bcfc23aecc477c97d487eebca.png)
 
-### Product Listing Page (PLP) template selection criteria
+### PLP template selection criteria
+
+PLP templates are selected in FastStore based on URL slug matching. Templates defined at a department (e.g., `/office`) are always inherited by all subcategories (e.g., `/office/chairs`, `/office/desks`), unless a more specific template exists for that subcategory.
```

Comment:
suggestion: I believe it's a bit repetitive, since it's explained in the 2nd point, but if you think it's best to give more attention to this part, that's okay.
```suggestion
PLP templates are selected in FastStore based on URL slug matching. 
```